TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Base.java

```diff
@@ -138,19 +153,17 @@ public void onDisconnect() {
     public void onMessage(Message message, Communicatable sender) {
         MessageType type = message.getType();
         Object data = message.getData();
-        switch (type) {
-            case BASE_RESOURCES:
-                Map map = (Map) data;
-                int usedRam = servers.stream().mapToInt((server) -> server.getGroup().getRam()).sum() + proxies.stream().mapToInt((proxy) -> proxy.getGroup().getRam()).sum();
-                int availableRam = Math.max(0, ((Number) map.get("freeRam")).intValue() - getKeepFreeRam());
-                setAvailableRam(Math.max(0, Math.min(availableRam, maxRam - usedRam)));
-                double cpuLoad = (Double) map.get("cpuLoad");
-                setCpuLoad(cpuLoad);
-                boolean ready = (boolean) map.get("ready") && cpuLoad <= getMaxCpuLoad();
-                setReady(ready);
-                break;
-            default:
-                sendMessage(message);
+        if (type == MessageType.BASE_RESOURCES) {
```

Comment:
Please leave this as it was since there could be more messages added.

Reply:
That's Intellij's auto reformatting feature